docs/sources/flow/getting-started/migrating-from-operator.md

```diff
@@ -0,0 +1,262 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/getting-started/migrating-from-operator/
+description: Migrating from Grafana Agent Operator to Grafana Agent Flow
+menuTitle: Migrate from Operator
+title: Migrating from Grafana Agent Operator to Grafana Agent Flow
+weight: 320
+---
+
+# Migrating from Grafana Agent Operator to Grafana Agent Flow
+
+With the release of flow, Grafana Agent Operator is no longer the recommended way to deploy Grafana Agent in Kubernetes. Some of the Operator functionality has been moved into Grafana Agent
+itself, and the remaining functionality has been replaced by our Helm Chart.
+
+- The Monitor types (`PodMonitor`, `ServiceMonitor`, `Probe`, and `LogsInstance`) are all supported natively by Grafana Agent in Flow mode. You no longer are 
+required to use the Operator to consume those CRDs for dynamic monitoring in your cluster.
+- The parts of the Operator that deploy the agent itself (`GrafanaAgent`, `MetricsInstance`, and `LogsInstance` CRDs) are deprecated. We now recommend
+operator users use the [Grafana Agent Helm Chart](https://grafana.com/docs/agent/latest/flow/setup/install/kubernetes/) to deploy the Agent directly to your clusters.
+
+This guide will provide some steps to get started with Grafana Agent for users coming from Grafana Agent Operator.
+
+## Deploy Grafana Agent with Helm
+
+1. You will need to create a `values.yaml` file, which contains options for how to deploy your agent. You may start with the [default values](https://github.com/grafana/agent/blob/main/operations/helm/charts/grafana-agent/values.yaml) and customize as you see fit, or start with this snippet, which should be a good starting point for what the operator does:
+
+    ```yaml
+    agent:
+      mode: 'flow'
+      configMap:
+        create: true
+      clustering:
+        enabled: true
+    controller:
+      type: 'statefulset'
+    replicas: 2
+    crds:
+      create: false
+    ```
+
+    This config will deploy Grafana Agent as a `StatefulSet` using the built in [clustering](https://grafana.com/docs/agent/latest/flow/concepts/clustering/) functionality to allow distributing scrapes across all Agent Pods. 
```

Comment:
Would be this most common scenario when deploying the agent with helm?

Reply:
We don't have a good story for deploying shards with helm, so the choices are really "clustering" or "just one instance".

Given the state of clustering (I love it), I'd like to proceed like this, but if we have serious stability concerns, we can discuss.